srcpkgs/libwaylandpp/template

```diff
@@ -13,8 +13,10 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://github.com/NilsBrause/waylandpp"
 distfiles="https://github.com/NilsBrause/waylandpp/archive/${version}.tar.gz"
-checksum=e7f486165d3568c3558b5c7099133aea4a285b82820eeafad329fc10271c654d
+checksum=b20b45917382c6b87e9380130c9a1a1c563da2f498de5830df12fbce326dd9f5
 subpackages="libwaylandpp-doc wayland-scanner++ libwaylandpp-devel"
+# cstdint defines std::uint32_t types
+CXXFLAGS="-include cstdint"
```

Comment:
I'm pretty sure this didn't work in a single file which is why I added it here. There are a few failures without it and I ended up adding it to so many things I just said f it.

Same goes for the kodi packages, it will bomb out in config if you add it there. Forget if it was for glib or musl or both but.

Reply:
https://git.alpinelinux.org/aports/tree/community/kodi/gcc13.patch

Reply:
also https://git.alpinelinux.org/aports/tree/community/waylandpp/gcc13.patch

Reply:
ya pulled that too. even if it builds for me may try to go to bed. guessing qt6 will block this from building til the morning at least anyway

Reply:
built. thanks!

guess ill wait til morning though to re-push and build. hope qt6 is sync'd by then